README.md

```diff
@@ -67,7 +67,7 @@ Fluid.app is macOS only. Please do contribute instructions for other platforms.
   * Whitelist:
     * Allow browsing to any URL
   * Left:
-    * Enter the Remit URL as home page, including the `auth_key` parameter. E.g. `https://MY-REMIT.herokuapp.com/?auth_key=MY_AUTH_KEY`
+    * Enter the Remit URL as home page, including the `auth_key` parameter. E.g. `https://MY-REMIT.herokuapp.com/?auth_key=MY_AUTH_KEY`. When running on Heroku, `heroku config:get --app ex-remit AUTH_KEY` gets you a value for MY_AUTH_KEY.
```

Comment:
These are generic instructions intended for any company. Note how we use MY-REMIT in the URL.

Reply:
Q: Do you mean "we prefer no change"? 

What I'll do is I'll close this PR.